libs/shared/storyblok-sdk/src/api/queries/common/page.graphql

```diff
@@ -16,6 +16,7 @@ query pageItem($slug: ID!, $relations: String) {
       component
       title
       description
+      keywords
```

Comment:
Same, why `keywords`?

Reply:
Same here, someone add `keywords` to `content type` so during the build the type validation failed as they were not included in local graphql schema but present in Storyblok.